samples/QuickBooks.gs

```diff
@@ -0,0 +1,90 @@
+/**
+ * Populate constants with data specific to your application,
+ * found at https://developer.intuit.com/v2/ui#/app/appdetail/XXXXXXX
+ */
+var CLIENT_ID = '...';
+var CLIENT_SECRET = '...';
+
+// URLS found at Quickbooks Devloper Portal: https://developer.intuit.com
+var BASE_AUTH_URL = 'https://appcenter.intuit.com/connect/oauth2';
```

Comment:
Optional: In the other samples we typically just inline these values in `getService()`, since dev's don't need to set them.

Reply:
Done!